rpm/poppler.spec

```diff
@@ -29,6 +29,8 @@ BuildRequires:  boost-devel >= 1.71.0
 
 Patch1: 0001-Revert-Require-the-newer-qt5-provided-by-the-new-bas.patch
 Patch2: 0002-Don-t-build-qt5-demos-tests.patch
+Patch3: 0003-Revert-Async-api-for-certificate-validation.patch
+Patch4: 0004-Use-old-way-to-get-seconds-since-epoch.patch
```

Comment:
Could consider merging this to the first patch, it's more or less about the same thing. Either way.

Reply:
Reshuffled and merged.